packages/lib-classifier/src/plugins/tasks/Registry.js

```diff
@@ -0,0 +1,29 @@
+class Registry {
+  constructor () {
+    this.register = {}
```

Comment:
Maybe this can be a set rather than an object? See my comment below about uniqueness.

Reply:
Sounds good to me. I’ll do some reading. When I wrote this class, I thought I was basically creating a Map that errors if you try to set an existing key.